app/src/main/java/nerd/tuxmobil/fahrplan/congress/preferences/AlarmTonePreference.kt

```diff
@@ -0,0 +1,62 @@
+package nerd.tuxmobil.fahrplan.congress.preferences
+
+import android.content.Context
+import android.media.RingtoneManager
+import android.net.Uri
+import android.provider.Settings
+import android.util.AttributeSet
+import androidx.preference.Preference
+
+/**
+ * Represents an alarm tone preference.
```

Comment:
I moved the alarm tone functionality into this class. See 23b10ca4822a133ed18915842987e132b4a9205a. I do not use `onClick` yet because I could not retrieve an `activity` from within the `Preference`.